platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManagerMBean.java

```diff
@@ -34,4 +35,11 @@
      * @throws MigrationException thrown if one or more Configurations couldn't be exported
      */
     Collection<MigrationWarning> export(@NotNull String exportDirectory) throws MigrationException;
+
+    /**
+     * Get a collection of the migratables available for export
+     *
+     * @return Collection of type Describable
+     */
+    Collection<Describable> getInfo();
```

Comment:
:confused: But our `ConfigurationMigrationManager` implements **both** our service contract and the MBean. This sounds like a conversation for the office because I'm lost :wink: